core/src/actions.rs

```diff
@@ -62,41 +62,46 @@ fn execute_refresh_action(action: RefreshAction) -> Result<(), ActionExecError>
     Ok(())
 }
 
+fn execute_publish_datapoint_action(action: PublishDataPointAction) -> Result<(), ActionExecError> {
+    let _tx_id = sign_and_submit_transaction(&action.tx)?;
+    Ok(())
+}
+
 impl OraclePool {
-    /// Generates and submits the "Commit Datapoint" action tx
-    pub fn action_commit_datapoint(&self, datapoint: u64) -> Result<String, StageError> {
-        let parameters = PoolParameters::new();
-        let mut req = json::parse(BASIC_TRANSACTION_SEND_REQUEST).unwrap();
+    // /// Generates and submits the "Commit Datapoint" action tx
+    // pub fn action_commit_datapoint(&self, datapoint: u64) -> Result<String, StageError> {
```

Comment:
I think we can remove this code since the oracle now allows for publishing of the first datapoint.